assets/sass/_typography.scss

```diff
@@ -285,30 +285,31 @@ h2 {
   font-size: rem(32);
   line-height: $small-leading;
   margin-top: $base-spacing;
+  margin-bottom: 1em;
 
   @include media($tablet) {
-    font-size: rem(44);
+    font-size: rem(40);
   }
 }
 
 h3 {
   font-size: rem(24);
   font-weight: $heading-font-weight;
+  margin-top: 1.4em;
```

Comment:
I’m going to be switching these back to our spacing vars (with division/multiplication to get the same effect).

If our spacing vars aren’t ideal then I want to revisit those, rather than add (seemingly?) arbitrary values for spacing — not manageable in the long run.